### ISSUE TITLE
Use rems for font-size

### ISSUE THREAD

**Issue by minimalsm** [ethereum/ethereum-org-website#5215]
**Is your feature request related to a problem? Please describe.**

We have a lot of `font-size` declarations that are still using pxs. This is an accessibility problem as [using px for `font-size` doesn't provide support to users who have made text size adjustments in their browsers settings](https://css-tricks.com/is-it-better-to-use-ems-rems-than-px-for-font-size/).

**Describe the solution you'd like**

Refactor our codebase to use rem for `font-size` only.

**Describe alternatives you've considered**
None

**Additional context**

Example of the scaling problems caused when mixing rems and pxs.

<img width="1424" alt="Screenshot 2022-01-25 at 14 32 29" src="https://user-images.githubusercontent.com/62268199/150996397-b6fdc298-ae8e-4ee7-a129-5e64f0e8c5e8.png">

More context on px vs rem: https://www.24a11y.com/2019/pixels-vs-relative-units-in-css-why-its-still-a-big-deal/

**Want to contribute?**

We love contributions from the Ethereum community! Please comment on an issue if you're interested in helping out with a PR.

**Comment by SarveshLimaye** [comment]
@minimalsm can I work on this issue?

**Comment by luisbeqja** [comment]
@SarveshLimaye  Im doing

**Comment by minimalsm** [comment]
Hey @SarveshLimaye thanks for offering, looks like @luisbeqja has already done this work.

Thanks for the PR @luisbeqja 💪. In future please comment on the issue before starting work. If someone else has offered to work on an issue before you they'd be given the opportunity first.

**Comment by luisbeqja** [comment]
@minimalsm  oh sorry and thank you so much for the advice, it was my first contribution 💪🏼
